pkg/commands/process/settings/rules/javascript/express/default_cookie_config.yml

```diff
@@ -0,0 +1,88 @@
+patterns:
+  - pattern: |
+      {
+        cookie: $<!>$<HASH_CONTENT>
+      }
+    filters:
+      - either:
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_name_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_expiry_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_path_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_domain_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_secure_attribute
+  - pattern: |
+      cookieSession($<!>$<HASH_CONTENT>)
+    filters:
+      - either:
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_name_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_expiry_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_path_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_domain_attribute
+        - not:
+            variable: HASH_CONTENT
+            detection: express_default_cookie_config_secure_attribute
+languages:
+  - javascript
+auxiliary:
+  - id: express_default_cookie_config_name_attribute
+    patterns:
+    - |
+      { $<...>name: $<_>$<...> }
```

Comment:
The `$<...>`s should be unnecessary inside an object/hash. If you do need them to make it work, we probably need to fix something elsewhere.

Reply:
Thanks @didroe - looks like it's working fine without these  🙌